en/orm/saving-data.rst

```diff
@@ -874,8 +874,8 @@ the singular, underscored version of the association name. For example::
 Saving HasOne Associations
 --------------------------
 
-When saving hasOne associations, the ORM expects a single nested entity at the
-singular, underscored version of the association name. For example::
+When saving hasOne associations, the ORM expects a single nested entity named with the
+singular, lowercase version of the association name. For example::
```

Comment:
CakePHP at its core often works on inflected names.

http://book.cakephp.org/3.0/en/search.html?check_keywords=yes&area=default&q=inflect

Using its built-in functionality of the [Inflector](http://book.cakephp.org/3.0/en/core-libraries/inflector.html).

I imagine a table on the Inflector documentation page, displaying all variants at a glance.

So in case of the "Saving BelongsTo Associations" documentation, we could link to this table from the sentence "the singular, underscored version of the association name".